eng/Subsets.props

```
@@ -80,6 +80,7 @@
     <SubsetName Include="Clr.CoreLib" Description="The managed System.Private.CoreLib library for CoreCLR." />
     <SubsetName Include="Clr.NativeCoreLib" Description="Run crossgen on System.Private.CoreLib library for CoreCLR." />
     <SubsetName Include="Clr.Tools" Description="Managed tools that support CoreCLR development and testing." />
+    <SubsetName Include="Clr.DiagTools" Description="Diagnostics tools needed to build CoreCLR." />
```

What do you think of?
```suggestion
    <SubsetName Include="Clr.BuildTools" Description="Diagnostics tools needed to build CoreCLR." />
```

I am guessing the high order bit for the developers working in clr is that you now need this tool in order to build the runtime, regardless whether you care about using the diagnostics or not.

---

If that were the case I would eliminate the new subset and add it to the `Clr.Tools` subset.

---

The Clr.Tools subset has to be built after CoreCLR because crossgen2 need the native build to have completed.

---

The Tools subset aren't required for building as I understood it. I'd imagine that is an important distinction if you are trying to build the minimal amount necessary to produce the runtime.

---

I have been thinking we should have prerequisite subsets.

---

I do like changing the new subset to "BuildTools"  because it does describe it more generally.  Eventually building DacTableGen should be moved to this subset like Jeremy suggested.

I'll rename it.